LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
```

Comment:
Just curious why this license was the one you picked.  This appears to be a very code oriented license and I wonder if something like CC-BY might be more appropriate for this particular thing.

Reply:
I didn't consider CC-BY, as I was mostly looking for a repository to fork that would help me get started. 

I certainly see your point and think CC-BY would be appropriate - either instead of or in addition to a software license.

but see also https://choosealicense.com/non-software/

> Open source software licenses can be also used for non-software works and are often the best choice, especially when the works in question can be edited and versioned as source.